webpack.config.js

```diff
@@ -9,15 +8,15 @@ module.exports = {
       {
         test: /\.ts?$/,
         use: "ts-loader",
-        exclude: /node_modules/,
+        // set excludes in tsconfig.json instead, b/c we use ts-loader
```

Comment:
nit: Writing comments about code that isn't present can be confusing.  I'd remove it if it doesn't feel critical or else add more context ("Normally we'd have an "excludes" here that ... but ...").